src/CommunityToolkit.Maui.MediaElement/Extensions/AndroidPermissions.android.cs

```diff
@@ -0,0 +1,16 @@
+﻿using static Microsoft.Maui.ApplicationModel.Permissions;
+
+namespace CommunityToolkit.Maui.Extensions;
+sealed class AndroidPermissions : BasePlatformPermission
+{
+	public override (string androidPermission, bool isRuntime)[] RequiredPermissions =>
+		new List<(string androidPermission, bool isRuntime)>
```

Comment:
this list will alwalys be same values, let's cache it as an array

Reply:
I'm honestly perplexed at how to do this. Would appreciate any suggestions.